src/components/data/DataTable.js

```diff
@@ -319,132 +447,7 @@ const DataTable = props => {
               initialY,
               sort,
               numFixedColumns: visibleColumns.length > 0 ? 2 : 0,
```

Comment:
this causes a page-breaking JS error if: 1) the selection columns are hidden; and 2) the table has <=2 columns.

The "2" in here needs to be conditional, I believe.
```suggestion
              numFixedColumns: visibleColumns.length > 0 ? (dataProvider.features.supportsRowSelection ? 2 : 1) : 0,
```

Reply:
Added -- thanks for the help here.